android-test/build.gradle

```diff
@@ -1,6 +1,15 @@
 apply plugin: 'com.android.library'
 apply plugin: 'org.jetbrains.kotlin.android'
 
+repositories {
+  jcenter {
+    // Required for a dependency of Android lint.
```

Comment:
This feels like it should include a link to a tracking bug somewhere.

Why does this code care about Android lint’s dependencies?!

Reply:
I hope this is right place https://issuetracker.google.com/issues/139207720